packages/ui-components/src/components/getting-started/getting-started-step.ts

```diff
@@ -0,0 +1,19 @@
+import { property } from 'lit/decorators.js';
+import { BaseElement } from '../../base';
+import { html } from 'lit';
+
+export class GettingStartedStep extends BaseElement {
+  @property({ type: String })
+  title = '';
+
+  render() {
+    return html`
+      <div class="getting-started-step text-white block">
```

Comment:
```suggestion
      <div class="text-white block">
```

Reply:
fixed in: https://github.com/JortWillemsen/stryker-dashboard-redesign/pull/20